### DISCUSSION TITLE
ART 1.11.1

### DISCUSSION THREAD

**Discussion by beat-buesser** [Trusted-AI/adversarial-robustness-toolbox#1846] [Announcements]
This release of ART 1.11.1 provides updates to ART 1.11.

# Added

[None]

# Changed

[None]

# Removed

[None] 

# Fixed

- Fixed unecessary check for targeted attacks in `AdversarialPatch` and delegated check to framework-specific implementations (#1768)
- Fixed missing transfer to device in `AdversarialPatchPyTorch.apply_patch()` (#1771)
- Fix redundant call to detach().cpu().numpy() in `PyTorchClassifier.predict()` (#1785)
- Fixed `art.utils.random_sphere()` for `norm=1` to sample uniformly in the L1 ball (#1802)
- Fixed PyTorch detach() call on Numpy arrays in ` PyTorchRegressor` (#1824)
- Fixed probability check for multi-dimensional arrays + out of bounds error in binning in the pointwise differential training privacy metric `PDTP` (#1825)
- Fixed learning rate decay in `ElasticNet` evasion attack (#1833)

<hr /><em>This discussion was created from the release <a href='https://github.com/Trusted-AI/adversarial-robustness-toolbox/releases/tag/1.11.1'>ART 1.11.1</a>.</em>
